lib/kitchen/provisioner/install_win.erb

```diff
@@ -1,8 +1,8 @@
 <%=
 salt_install = config[:salt_install]
 salt_url = config[:salt_bootstrap_url]
-bootstrap_options = config[:salt_bootstrap_options]
 salt_version = config[:salt_version]
+bootstrap_options = config[:salt_bootstrap_options].gsub(/%s/, salt_version)
```

Comment:
can we do this instead?

https://github.com/saltstack/kitchen-salt/blob/d0bd55c51837a26b2d6369f48d107ae8cf90239e/lib/kitchen/provisioner/install.erb#L5

Reply:
@gtmanfred happily done. I also merged in the latest changes from master to ensure a smooth merge.